pkg/k8shandler/dashboards.go

```diff
@@ -0,0 +1,45 @@
+package k8shandler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+)
+
+const (
+	clusterLoggingDashboardFile = "dashboards/openshift-logging-dashboard.json"
+)
+
+// CreateOrUpdateDashboards reconciles metrics dashboards component for cluster logging
+func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateDashboards() (err error) {
+	if err = clusterRequest.createOrUpdateClusterLoggingDashboard(); err != nil {
+		return
+	}
+	return nil
+}
+
+// createOrUpdateClusterLoggingDashboard creates/updates the cluster logging dashboard ConfigMap
+func (clusterRequest *ClusterLoggingRequest) createOrUpdateClusterLoggingDashboard() error {
+	b, err := ioutil.ReadFile(path.Join(utils.GetShareDir(), clusterLoggingDashboardFile))
+	if err != nil {
+		return fmt.Errorf("failed to read dashboard file: %w", err)
+	}
+
+	cm := NewConfigMap("grafana-dashboard-cluster-logging",
```

Comment:
consider naming this just "cluster-logging-dashboard" or "dashboard-cluster-logging"

Reply:
All dashboards in that namespace are formatted like this `grafana-dashboard-<name>`